src/splits.rs

```diff
@@ -430,20 +434,21 @@ pub enum Split {
     /// Rescued Flea Underworks Explosions (Flea)
     ///
     /// Splits after rescuing flea in Under_21
-    SavedFleaUnderworksExplosions,
+    #[alias = "SavedFleaUnderworksExplosions"]
+    SavedFleaUnderworksCauldron,
```

Comment:
This only renames the variant key `SavedFleaUnderworksCauldron`. Did you also mean to rename the actual split name from `Rescued Flea Underworks Explosions (Flea)` to `Rescued Flea Underworks Cauldron (Flea)`?